app/views/courses/locations.html.erb

```diff
@@ -0,0 +1,33 @@
+<%= content_for :page_title, "Locations - #{course.name_and_code}" %>
+
+<% content_for :before_content do %>
+  <%= govuk_back_link_to(provider_course_path(course.provider.provider_code, course.course_code)) %>
+<% end %>
+
+<h1 class="govuk-heading-xl">
+  <span class="govuk-caption-xl"><%= course.name_and_code %></span>
+  Locations
+</h1>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= form_for course, url: publish_provider_course_path(course.provider.provider_code, course.course_code), method: :post do |f| %>
+      <div class="govuk-form-group">
+        <div class="govuk-checkboxes">
+          <%= f.fields_for :site_statuses, @provider.sites do |sf| %>
+            <div class="govuk-checkboxes__item">
+              <%= f.check_box(sf.object.id, checked: false, class: 'govuk-checkboxes__input') %>
+              <%= f.label(sf.object.id, sf.object.location_name, class: 'govuk-label govuk-checkboxes__label') %>
+            </div>
+          <% end %>
+        </div>
+      </div>
+
+      <%= f.submit "Save", class: "govuk-button", data: { qa: 'course__save' } %>
```

Comment:
If the course is already published, this should say "Save and publish changes" otherwise it's just "Save"